MapboxCoreNavigationTests/PassiveLocationDataSourceTests.swift

```diff
@@ -70,7 +70,9 @@ class PassiveLocationDataSourceTests: XCTestCase {
         wait(for: [startingExpectation], timeout: 2)
         
         // FIXME: Starting the location manager should automatically configure the navigator.
+        XCTAssertFalse(locationManager.isConfigured)
         XCTAssertNoThrow(try locationManager.configureNavigator(withTilesVersion: "1234"))
+        XCTAssertTrue(locationManager.isConfigured)
```

Comment:
This test is currently disabled – #2579 reenables it after reworking it a bit – but I verified locally that the new assertions pass.